RELEASE_NOTES.md

```diff
@@ -1,66 +1,48 @@
-
-Added a getDeviceAllocator function that allows users to get a DeviceAllocator object from the kernel without explicitly passing the allocator to the kernel first.
-
-Added a reset function to the DeviceAllocator so that old data can be rewritten.
-
-Expose "PREFETCH" operations registered with the MemoryOperationRegistry with a new ResourceManager::prefetch method.
-
-Build Doxygen documentation on ReadTheDocs.
-
-Add `-rdynamic` to backtrace example
-
-Add CI job with interprocess shared memory and CUDA
-
-Add CI containers to allow for gcc{7,8,9}, clang{11,12}, and nvcc{10,11}
-
-Add CI to use C++14 version of XL Compiler
-
-Add CI to check pools work with DEVICE_CONST memory
-
-Add Umpire Allocator profile comparison benchmark
-
-Add replay `--introspection-off` option for strategies
-
-Fix warning caused by ignoring posix_memalign return value.
-
-Fixed a cmake install config issue so that now users can find a package of Umpire with a version constraint.
-
-Fix `ResourceManager::isAllocator` to work for resources
-
-Fix comparison operators for `TypedAllocators`. Move the operators into the same namespace as `TypedAllocator` for ADL instead of leaving in the global namespace.
-
-Fix host and device Allocator ID overlap
-
-Fix return value from get_process_highwater_mark by correctly treating VmHWM as kB
-
-Remove null and zero-byte pool from list of valid allocators
+## Changes Impacting Builds
+This release of Umpire contains new build requirements including:
+
+- C++14 is now required to build Umpire
+- `cmake` version 3.14 or later is required
+
+## Changes Impacting C/Fortran
+- The `cmake` object library for `C/FORTRAN` interface has been reorganized.
+  (**NOTE**: *This is a breaking change since the include paths are now different.*)
+
+## New Interfaces
+- Added a `getDeviceAllocator` function that allows users to get a `DeviceAllocator` object from the kernel without explicitly passing the allocator to the kernel first.
+- Added a `reset` function to the `DeviceAllocator` so that old data can be rewritten.
+- Expose `PREFETCH` operations registered with the `MemoryOperationRegistry` with a new `ResourceManager::prefetch` method.
+
+## Removed Interfaces
+The following functions previously marked as deprecated have now been removed:
+
+- `DynamicPoolMap` and `DynamicPool` aliases removed
+- `registerAllocator` and `isAllocatorRegistered` removed
+
+## Fixes
+- Fixed a cmake install config issue so that now users can find a package of Umpire with a version constraint.
+- Fix `ResourceManager::isAllocator` to work for resources
+- Fix comparison operators for `TypedAllocators`
+- Fix host and device Allocator ID overlap
+- Remove null and zero-byte pool from list of valid allocators
+
+## New Configuration Options
+- The `UMPIRE_ENABLE_DEVICE_ALLOCATOR` option was added to control whether or not the DeviceAllocator class is included in the library.  The default is "On".
```

Comment:
I think I changed the default to be off

Reply:
Oh really?  Oops

Reply:
Fixed

Reply:
Thanks! Yea, I just confirmed it is off by default. I think it was originally On by default, but then when I added some of my DA updates, that caused some problems. (So I don't think it was well documented that it was subsequently turned Off by default!)